Rubberduck.Core/UI/Command/VersionCheckCommand.cs

```diff
@@ -53,40 +53,31 @@ protected override async void OnExecute(object parameter)
             Logger.Info("Executing version check...");
 
             var tokenSource = new CancellationTokenSource(TimeSpan.FromSeconds(5));
-            await _versionCheck
-                .GetLatestVersionAsync(settings, tokenSource.Token)
-                .ContinueWith(t =>
```

Comment:
This was the problem; the service's return value should have been awaited here, not task-continued like this.